### PR TITLE
Fixes #4791 - Update Favorite button on MyShots Page

### LINKED ISSUE #4791: [FxA] Update Favorite button on shot page
This is under-defined in the spec on both the shot page and the my shots page. We need a better overall interaction pattern to indicate favorite-ing shots.

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: static/img/icon-heart.svg [hunk 1]
```diff
@@ -1 +1,16 @@
...
+    <g id="FxA" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="#4791" transform="translate(-1113.000000, -303.000000)" fill="#B5007F" fill-rule="nonzero">
+            <g id="Group-8" transform="translate(972.000000, 277.000000)">
+                <g id="Heart" transform="translate(141.000000, 26.000000)">
+                    <path d="M20.7274507,1.36450775 C18.2705306,-0.715670426 14.6165345,-0.341505616 12.3613688,1.97029839 L11.4781329,2.87453002 L10.594897,1.97029839 C8.34421479,-0.341505616 4.68573525,-0.715670426 2.22881518,1.36450775 C-0.586779367,3.75203559 -0.734732583,8.03711354 1.78495553,10.6250868 L10.4603941,19.5248641 C11.020823,20.0994743 11.9309594,20.0994743 12.4913883,19.5248641 L21.1668269,10.6250868 C23.6909984,8.03711354 23.5430452,3.75203559 20.7274507,1.36450775 Z" id="Shape-Copy"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (chenba) [inline static/img/icon-heart.svg]: Need to run the new svgs through svgo.

**Comment by chenba** [comment]
The button text color on My Shots need to be updated.  And according to https://github.com/mozilla-services/screenshots/issues/4791#issuecomment-422634013 the icons have been updated, so we should pull those in.

**Comment by punamdahiya** [comment]
@chenba PR is updated with new UX for favorite icon on my shots and shot page. Please review. Thanks!

File: server/src/pages/shot/view.js [hunk 2]
```diff
@@ -341,10 +341,10 @@ class Body extends React.Component {
     let editButton = null;
     const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
     const activeFavClass = this.props.expireTime ? "" : "is-fav";
-    const shouldShow = this.props.isFxaAuthenticated ? "" : "hidden";
+    const shouldDisable = this.props.isFxaAuthenticated ? "" : "disabled";
 
     favoriteShotButton = <div className="favorite-shot-button"><Localized id="shotPagefavoriteButton">
-      <button className={`nav-button ${shouldShow} `} onClick={this.onClickFavorite.bind(this)}>
+      <button className={`nav-button ${shouldDisable} `} onClick={this.onClickFavorite.bind(this)}>
```
Comment (chenba) [inline server/src/pages/shot/view.js]: This should use the `disabled` attribute, not a class.

**Comment by punamdahiya** [comment]
PR updated with feedback, thanks

**Pull request merged**